testsuite/features/secondary/min_salt_migration.feature

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+# This feature depends on 
+# - minkvm_guests.feature which has already configured the KVM Minion to be a virtualization host.
+# - srv_sync_channels.feature which syncs SLES 15 SP4 and the client tools 
+
+@scope_salt
+@scope_salt_ssh
+@scope_virtualization
+@virthost_kvm
+Feature: Migrate Salt to bundled Salt on a nested Minion VM
+
+  Scenario: Log in as admin user in the Salt migration context
+    Given I am authorized for the "Admin" section
+
+  Scenario: Show the KVM host system overview
+    Given I am on the Systems overview page of this "kvm_server"
+
+  Scenario: Prepare a SLES KVM test VM and list it in the Salt migration context
+    When I delete default virtual network on "kvm_server"
+    And I create salt-sles virtual network on "kvm_server"
+    And I delete default virtual storage pool on "kvm_server"
+    And I create salt-sles virtual storage pool on "kvm_server"
+    And I create a sles virtual machine named "salt-sles-migration-test" on "kvm_server"
+    And I follow "Virtualization" in the content area
+    And I wait until I see "salt-sles-migration-test" text
+
+  Scenario: Start the SLES 15 KVM VM in the Salt migration context
+    Given I follow "Virtualization" in the content area
+    When I click on "Start" in row "salt-sles-migration-test"
+    Then I should see "salt-sles-migration-test" virtual machine running on "kvm_server"
+    # We need a small delay to give cloud init some time to do its magic
+    And I wait for "30" seconds
```

Comment:
Isn't there something specific we can wait for?

Reply:
At this point I think not. I have the other step above in line 32 to wait until the VM is powered on using `virsh domstate` but there cloud init has not been finished, yet.

Reply:
The checks for cloud init to be finished is done in the next scenario where is checks if certain configuration files have certain entries.

Reply:
Maybe you can use something similar to what we use here?

```
    And I click on "Create"
    And I wait until I see "Hosted Virtual Systems" text
    And I wait 180 seconds until the event is picked up and 300 seconds until the event "Creates a virtual domain: test-vm2" is completed
    And I follow "Virtualization" in the content area
```

Reply:
There is a cloud init [command](https://cloudinit.readthedocs.io/en/latest/topics/cli.html#cli-status) called `cloud-init status` that can be executed on the nested VM to check if cloud init is finished.
```bash
$ cloud-init status
status: running


$ cloud-init status
status: done
```

Reply:
yeah would be nice to use instead of a fixed wait

Reply:
Honestly, if you could do it now, would be better. It's important we avoid those waits like the plague.

Reply:
Ok, then I try to do it with `cloud-init status` and see if this works.